packages/pluggableWidgets/switch-web/src/Switch.tsx

```diff
@@ -0,0 +1,55 @@
+import { createElement, FunctionComponent, KeyboardEvent, MouseEvent, useCallback } from "react";
+import { executeAction, isAvailable } from "@mendix/piw-utils-internal";
+import { ValueStatus } from "mendix";
+import { Switch as SwitchComponent } from "./components/Switch";
+
+import { SwitchContainerProps } from "../typings/SwitchProps";
+import "./ui/switch-main.scss";
+
+export const Switch: FunctionComponent<SwitchContainerProps> = props => {
```

Comment:
Can we export a function instead and remove some symbols?

Reply:
Good one, thanks. Addressed 👍